### DISCUSSION TITLE
Unable to display Mermaid Diagram

### DISCUSSION THREAD

**Discussion by javianng** [shuding/nextra#3591] [Q&A]
Upon copying the code from the documentation page that is intended to render the Mermaid diagram, I was provided with a code block. I have cloned the repository from the documentation template at https://github.com/shuding/nextra-docs-template.

### Rendered Markdown

<img width="734" alt="image" src="https://github.com/user-attachments/assets/ebd691b2-dcdf-4d31-98da-9fbc602c37d5">

### Code

<img width="585" alt="image" src="https://github.com/user-attachments/assets/25495c29-efcb-4af2-bf22-f85f84ac5bb6">

**Comment by dimaMachina** [comment]
Use nextra v3

**Comment by MikeAndFriends** [comment]
Hi!
I'm using nextra v3 ("nextra": "^3.3.0"). Receive an error when trying to display a mermaid page (same content as above). The error received:

```
Unhandled Runtime Error

 Error: Copy to clipboard is not supported in this browser
SyntaxError: Unexpected token '{'

eval
[native code] (0:0)
<unknown>
file:///var/www/edo_dev.nextra.isayconsulting/.next/static/chunks/webpack.js (712:35)
__webpack_require__
file:///var/www/edo_dev.nextra.isayconsulting/.next/static/chunks/webpack.js (37:37)
fn
[native code] (0:0)
promiseReactionJob
[native code] (0:0)
Show ignored frames
```

Upon setup of site I already had a next.js project (using `npx create-next-app@latest` ) then did a `npm i next react react-dom nextra nextra-theme-docs` in the project folder. Any problems with that?

**Reply by MikeAndFriends** [reply]
sorry. seems to be a browser thing. works fine in chrome...
